core/tests/performance_tests/test_config.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Performance test config."""
+
+TEST_DATA = [{
```

Comment:
I'm not sure what you're suggesting @maitbayev. I think the point of this file is to separate out constants which control the verification steps of the performance tests such that they are easily tweakable and configurable. It's much cleaner to decide all pages in Oppia should now load in under 2 seconds and change it in this one location rather than going through and updating all tests page-by-page.